core/lib/presentation/utils/html_transformer/text/new_line_transformer.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:convert';
+
+import 'package:core/presentation/utils/html_transformer/base/text_transformer.dart';
+
+class NewLineTransformer extends TextTransformer {
+  const NewLineTransformer();
+
+  @override
+  String process(String text, HtmlEscape htmlEscape) {
+    return text.replaceAll('\n', '<br>');
```

Comment:
We should also pay attention to other line break tags like `\r\n` or `\r`

Reply:
Done